scenarios/kubernetes_build.py

```diff
@@ -71,6 +78,21 @@ def main(args):
     else:
         check('make', 'release')
     check('../release/push-build.sh', *push_build_args)
+    # log push-build location to path child jobs can find
+    # (gs://<shared-bucket>/$PULL_REFS/bazel-build-location.txt)
+    # first get build location
+    bucket = args.release or 'kubernetes-release-dev'
+    dest = 'ci'
+    if args.suffix:
+        dest += args.suffix
+    build_tar = tarfile.open("_output/release-tars/kubernetes.tar.gz")
+    latest = build_tar.getmember("kubernetes/version").read().strip()
+    gcs_build = 'gs://'+bucket+dest+latest
```

Comment:
this does not give you the right path - maybe use `os.path.join`

and I would suggest you add a unittest for both bazel and build scenario as well